common/tdx/verifier.c

```diff
@@ -150,8 +152,22 @@ static oe_result_t _fill_with_known_tdx_claims(
                             OE_TDX_ADDITIONAL_CLAIMS_COUNT)
         OE_RAISE(OE_INVALID_PARAMETER);
 
+    /* Default TDX quote version is 4 */
     tdx_quote = (tdx_quote_t*)quote;
-    tdx_report = &tdx_quote->report_body;
+    if (tdx_quote->version == 4)
+    {
+        tdx_report = &tdx_quote->report_body;
+    }
+    else if (tdx_quote->version == 5)
+    {
+        tdx_quote_v5 = (tdx_quote_v5_t*)quote;
+        if (tdx_quote_v5->type == 1)
```

Comment:
Can you add comment on what does each type mean?

Reply:
added